.github/workflows/linting.yaml

```diff
@@ -39,4 +39,4 @@ jobs:
           python setup.py build_cmake --inplace
 
       - name: Run pylint
-        run: python3 -m pylint --disable=R,W flamingpy/*.py flamingpy/benchmarks/*.py flamingpy/examples/*.py tests/unit/*.py
+        run: python3 -m pylint --disable=R,W flamingpy/**/*.py tests/**/*.py

```

Comment:
Does this notation include the files in flamingpy directory that are not in a subfolder?

Also the pylint test seems to pass with 10/10 -- is that suspicious?